types/webpack-dev-middleware/index.d.ts

```diff
@@ -9,7 +9,7 @@ import * as webpack from 'webpack';
 export = WebpackDevMiddleware;
 
 declare function WebpackDevMiddleware(
-	compiler: webpack.ICompiler,
+	compiler: webpack.Compiler | webpack.MultiCompiler,
```

Comment:
@alan-agius4 could you explain what you are trying to do here? MultiCompiler implements ICompiler.

Reply:
But `MultipleCompiler` is not assignable to `Compiler` because they are different and some properties are missing in MultipleCompiler

```
Argument of type 'MultiCompiler' is not assignable to parameter of type 'Compiler'.
  Property 'name' is missing in type 'MultiCompiler'.
const multiCompiler: webpack.MultiCompiler
```